script/entrypoint

```diff
@@ -19,10 +19,10 @@ fi
 
 if [ "$#" = 0 ]; then
     # Default to 20 workers because most of the tests are io-bound
-    exec pytest --black --docstyle --flake8 --mypy-ignore-missing-imports --numprocesses 20
+    make -f script/Makefile PYTESTFLAGS="--black --docstyle --flake8 --mypy-ignore-missing-imports --numprocesses 20"
```

Comment:
What if we left these invocations as-is and instead added a `conftest.py` in the `tests` directory that invokes the method for generating the sql files?

That would ensure that any time you run `pytest`, you're sure to have up-to-date generated SQL files, whether you are calling pytest through this entrypoint script or not. It would also allow us to avoid the complexity of adding a Makefile for the time being.

If you think we're better off sticking with a Makefile, would it make sense to put it at the top level of the respository? That's what I'm most used to seeing, so that you could clone the repo, `cd` into it, run `make`, and a good set of default tasks run to get you all set up.

In that case, the Makefile would look something like:

```
build: test

test: target/sql 
	./script/entrypoint

target/sql: sql/
	python script/generate_sql

```